### ISSUE TITLE
Accessibility issues with animated gifs

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity-support#19]
The animated gifs which have been used in Audacity Support break a number of accessibility guidelines.
See, for example, https://www.bbc.co.uk/gel/features/how-to-make-accessible-animated-images
Note in particular that the issue with blinking, moving, scrolling stuff is Level A: https://www.w3.org/TR/WCAG21/#pause-stop-hide
It may be easier just not to use them.

As a connected issue, if you export a page which includes an animated gif as a pdf, the image in the pdf is not animated. So if any information was only available in the animation, this will be lost in the pdf.

**Comment by LWinterberg** [comment]
GIFs should be removed now
